docker/pulp/Dockerfile

```diff
@@ -18,17 +19,26 @@ RUN mkdir -p /var/run/pulp \
     /var/lib/pulp/tmp
 
 COPY docker/pulp/requirements.txt /tmp/pulp/
+COPY docker/pulp/requirements_ansible_test.txt /tmp/ansible_test/
+
+RUN python3.6 -m venv "${ANSIBLE_TEST_VENV}" \
+    && source "${ANSIBLE_TEST_VENV}/bin/activate" \
+    && pip --no-cache-dir install -U \
+        'pip<19.0' \
+        wheel \
+    && cd /tmp/ansible_test \
+    && pip install --default-timeout 100 -r requirements_ansible_test.txt \
+    && deactivate
 
 RUN python3.6 -m venv "${PULP_VENV}" \
     && source "${PULP_VENV}/bin/activate" \
     && pip --no-cache-dir install -U \
         'pip<19.0' \
         wheel \
     && cd /tmp/pulp \
-    && pip install --default-timeout 100 -r requirements.txt \
+    && pip install --default-timeout 100 -r requirements.txt
 
-# FIXME(cutwater): This is temporary fix for ansible/galaxy-dev#141
-RUN chmod -R a+rwX /venv
+RUN chmod -R a+rwX "${ANSIBLE_TEST_VENV}"/lib64/python*
```

Comment:
Can you please confirm what actually ansible-test does write to this directory.
In case it's just config file, like you mentioned earlier, it doesn't need to be a volume.

Reply:
@cutwater the only thing I am aware of is writing `ansible.egg-info/PKG_INFO` into `.../lib64/python3.6/site-packages/`

Reply:
@cutwater PR has been updated and the venv was removed as a docker-compose persistent volume

Reply:
Please merge it with RUN on line 25.

Reply:
updated